components/telekonseling/telekonseling-link-section.tsx

```diff
@@ -0,0 +1,24 @@
+import { InternalPageSection } from "~/components/layout/page";
+import StackedLink from "~/components/layout/stacked-link";
+import { Telecounseling } from "~/lib/content/telekonseling";
+
+interface TelecounselingSectionProps {
+  telekonseling: Telecounseling;
+}
+
+export function TelecounselingSection(props: TelecounselingSectionProps) {
+  return (
+    <InternalPageSection className="flex-1 pb-6 space-y-8">
+      {props.telekonseling.supports.map((counseling, index) => (
+        <div key={index} className="space-y-4">
```

Comment:
```suggestion
      {props.telekonseling.supports.map((counseling) => (
        <div key={counseling.support} className="space-y-4">
```

Please [avoid using `index` as keys](https://reactjs.org/docs/lists-and-keys.html#keys).
<img width="744" alt="Screenshot 2021-08-17 at 22 30 43" src="https://user-images.githubusercontent.com/6315466/129745159-cddc1e08-7149-4b0c-892e-f187bbbe3148.png">